container-entrypoint.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: container-entrypoint
+  version: 0.1.0
+  epoch: 0
+  description: Simple entrypoint script for containers
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - busybox
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/wolfi-dev/container-entrypoint/archive/refs/tags/${{package.version}}.tar.gz
+      expected-sha512: 32cea7963406d75cb6023626ac6a5969a6178ae94a49cc0400c80ff223a8d3874b85e4a67a8276faaf0bd39e4ae26c2708f6b0f4aef1064020490cf79643fd8c
+  - runs: |
+      mkdir -p ${{targets.destdir}}/bin
+      cp entrypoint.sh ${{targets.destdir}}/bin/
```

Comment:
why not

```
- runs: |
    install -Dm755 entrypoint.sh ${{targets.destdir}}/bin/
```

?

Reply:
Because @rawlingsj didn't tell me to do that 😂 

There's no reason, I'll change it now.